generator/interfaces/gen.go

```diff
@@ -0,0 +1,167 @@
+package interfaces
+
+import (
+	"bytes"
+	"go/ast"
+	"go/printer"
+	"strings"
+
+	"github.com/devfile/api/generator/genutils"
+	"sigs.k8s.io/controller-tools/pkg/genall"
+	"sigs.k8s.io/controller-tools/pkg/markers"
+
+	"github.com/elliotchance/orderedmap"
+)
+
+//go:generate go run sigs.k8s.io/controller-tools/cmd/helpgen generate:headerFile=../header.go.txt,year=2020 paths=.
+
+var (
+	toplevelListMarker = markers.Must(markers.MakeDefinition("devfile:toplevellist", markers.DescribesField, struct{}{}))
+)
+
+type copyStructs struct {
```

Comment:
IDE tells it's used why then not remove it?

Reply:
Good catch. It's an old remainder of the original code I used to bootstrap this file.

Reply:
Fixed in commit https://github.com/devfile/api/pull/133/commits/54fc1dc3059653c84260c92fc82a09476742337c